app/controllers/protected_species_controller.rb

```diff
@@ -1,11 +1,17 @@
 class ProtectedSpeciesController < PreAuthController
   def index
     @species = policy_scope(Species.protected_species).includes(:species_order, :species_family).order("species_order.name, species_families.name, species.name")
```

Comment:
For a separate PR: You could move `.includes(:species_order, :species_family).order("species_order.name, species_families.name, species.name")` into the `protected_species` `scope`: 

```suggestion
    @species = policy_scope(Species.protected_species)
```